creation/interfaces-and-networks.md

```diff
@@ -736,6 +736,9 @@ spec:
 > **Note:** for some NICs (e.g. Mellanox), the kernel module needs to be
 > installed in the guest VM.
 
+> **Note:** Placement on dedicated CPUs can only be achieved if the Kubernetes CPU manager, is running on the SR-IOV capable workers.
```

Comment:
I believe that the comma after "manager" should not be there.

Reply:
thanks:)
Done.